zebra-chain/src/transaction/transparent.rs

```diff
@@ -80,8 +83,7 @@ pub enum TransparentInput {
 pub struct TransparentOutput {
     /// Transaction value.
     // At https://en.bitcoin.it/wiki/Protocol_documentation#tx, this is an i64.
-    // XXX refine to Amount ?
-    pub value: u64,
+    pub value: Amount<NonNegative>,
```

Comment:
Is there some canonical description of how Bitcoin does this kind of checking? If we do this, we reject negative values during parsing.  Is that fine, if we need exact compatibility with Zcashd? I think it is probably fine but I'm not sure.

Reply:
You'll have to reverse-engineer zcashd to get that information, I'm afraid. Please document it when you do!

Reply:
Okay, we should probably split this question out into a separate issue then and unblock this PR.

Reply:
#566